internal/configuration/sources/env/wireguard.go

```diff
@@ -14,6 +14,10 @@ func (s *Source) readWireguard() (wireguard settings.Wireguard, err error) {
 	wireguard.Interface = s.env.String("VPN_INTERFACE",
 		env.RetroKeys("WIREGUARD_INTERFACE"), env.ForceLowercase(false))
 	wireguard.Implementation = s.env.String("WIREGUARD_IMPLEMENTATION")
+	wireguard.PersistenKeepAlive, err = s.env.Int("WIREGUARD_IMPLEMENTATION", env.ForceLowercase(false))
```

Comment:
change the var name 😉 

```suggestion
	wireguard.PersistenKeepAlive, err = s.env.Duration("WIREGUARD_PERSISTENT_KEEP_ALIVE", env.ForceLowercase(false))
```